common/startup/startup.go

```diff
@@ -86,74 +80,44 @@ func GetBackend(credProvider provider.CredProvider, region, mysqlConnection, adm
 	return backend, nil
 }
 
-func GetLogic(backend backend.Backend, mysqlConnection, adminConnection string) (*logic.Logic, error) {
-	sqlAdmin, err := getSQLAdmin(adminConnection)
-	if err != nil {
-		return nil, err
-	}
-
-	tagData, err := getTagData(mysqlConnection)
-	if err != nil {
-		return nil, err
-	}
-
-	jobData, err := getJobData(mysqlConnection)
+func GetLogic(backend backend.Backend) (*logic.Logic, error) {
+	tagStore, err := getTagStore()
 	if err != nil {
 		return nil, err
 	}
 
-	return logic.NewLogic(sqlAdmin, tagData, jobData, backend), nil
-}
-
-func getSQLAdmin(adminConnection string) (data.SQLAdmin, error) {
-	var getAdminDataStore func() (data.AdminDataStore, error)
-	if adminConnection != "" {
-		os.Setenv("LAYER0_MYSQL_ADMIN_CONNECTION", adminConnection)
-		getAdminDataStore = func() (data.AdminDataStore, error) { return data.NewMySQLAdmin() }
-	} else {
-		getAdminDataStore = func() (data.AdminDataStore, error) { return data.NewSQLiteAdminDataStore() }
-	}
-
-	adminDataStore, err := getAdminDataStore()
+	jobStore, err := getJobStore()
 	if err != nil {
 		return nil, err
 	}
 
-	return data.NewSQLAdminLayer(adminDataStore), nil
+	return logic.NewLogic(tagStore, jobStore, backend), nil
 }
 
-func getTagData(mysqlConnection string) (data.TagData, error) {
-	var getTagDataStore func() (data.TagDataStore, error)
-	if mysqlConnection != "" {
-		os.Setenv("LAYER0_MYSQL_CONNECTION", mysqlConnection)
-		getTagDataStore = func() (data.TagDataStore, error) { return data.NewTagMySQLDataStore() }
-	} else {
-		getTagDataStore = func() (data.TagDataStore, error) { return data.NewTagSQLiteDataStore() }
-	}
+func getTagStore() (tag_store.TagStore, error) {
```

Comment:
[nit] this is calling a constructor rather than just `get`ting the `TagStore`. Is `getTagStore` still an appropriate name given what it's actually doing?